TGC/history/countries/LBN - Lebanon.txt

```diff
@@ -1,7 +1,9 @@
 capital = 913
-primary_culture = mashriqi
-culture = bedouin
-religion = sunni
+primary_culture = druzes
+culture = syriac
+culture = druzes
```

Comment:
Both primary and accepted?

Reply:
On account of when different colonizers/GPs reorganize Lebanon, they will also be changing primary culture, ergo there should be accepted culture status for the group that fell out of grace

eg. Druze => Syriac => Mashriqi

technically just for flavour, most groups in LBN should be accepting each other

Reply:
I'm not sure that's gonna work but we'll see.